libs/core/src/lib/fixed-card-layout/fixed-card-layout.component.ts

```diff
@@ -24,7 +24,7 @@ import {
 import { FocusKeyManager } from '@angular/cdk/a11y';
 import { CdkDrag, CdkDragDrop, CdkDragEnter } from '@angular/cdk/drag-drop';
 import { Subject, Subscription } from 'rxjs';
-import { debounceTime, delay, distinct, takeUntil } from 'rxjs/operators';
+import { debounceTime, distinct, filter, takeUntil } from 'rxjs/operators';
```

Comment:
Unintended changes on this file

Reply:
Due to the merge did earlier, other commits were appended. I made undo of that merge.